lib/apple/index.js

```diff
@@ -89,6 +89,10 @@ exports.verifyPayment = function (payment, cb) {
 		});
 	}
 
+	if (typeof payment.secret !== 'undefined') {
```

Comment:
Probably just a habit of mine, because while `payment.secret !== undefined` works, stuff like `payment !== undefined` doesn't :). I can change that if you want.